### PR TITLE
Add BIP39 implementation reference

### PR DESCRIPTION
**Pull request by hujiulong**
Add a reference to the JS implementation

### CODE REVIEW

File: bip-0039.mediawiki [hunk 1]
```diff
@@ -168,6 +168,7 @@ Haskell:
 JavaScript:
 * https://github.com/bitpay/bitcore/tree/master/packages/bitcore-mnemonic
 * https://github.com/bitcoinjs/bip39 (used by [[https://github.com/blockchain/My-Wallet-V3/blob/v3.8.0/src/hd-wallet.js#L121-L146|blockchain.info]])
+* https://github.com/hujiulong/web-bip39
```
Comment (rezaeghba01) [inline bip-0039.mediawiki:171]:
```suggestion
* https://github.com/hujiulong/web-bip39
```

**Review by rezaeghba01 — CHANGES_REQUESTED** [review]

File: bip-0039.mediawiki (hunk 1 above)
Comment (Yuttana12wr) [reply bip-0039.mediawiki:171]: 2

**Comment by luke-jr** [comment]
@slush0 @prusnak @voisine @ebfull

**Comment by prusnak** [comment]
ACK

**Pull request merged**